cinema/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=64)
+    description = models.TextField(null=True, blank=True)
+    duration = models.IntegerField()
+
+    class Meta:
+        verbose_name_plural = "movies"
```

Comment:
are you sure that it is not the same as the default one?

Reply:
Fixed